configure.ac

```diff
@@ -25,6 +25,8 @@ AC_SYS_LARGEFILE
 AC_CHECK_LIB([popt],[poptParseArgvString],,
   AC_MSG_ERROR([libpopt required but not found]))
 
+AC_CHECK_FUNCS([asprintf fork madvise qsort strndup strptime vfork vsyslog])
```

Comment:
I do not understand the purpose of this commit.  It adds 69 lines and removes 44 lines despite it does not introduce any new feature, neither fixes any known bug?

To move things forward ... are you fine with merging #103 excluding this particular commit?

If this commit is really needed, we can discuss it in a separate PR.